query-service-impl/src/main/java/org/hypertrace/core/query/service/QueryRequestUtil.java

```diff
@@ -0,0 +1,55 @@
+package org.hypertrace.core.query.service;
+
+import org.hypertrace.core.query.service.api.ColumnIdentifier;
+import org.hypertrace.core.query.service.api.Expression;
+import org.hypertrace.core.query.service.api.LiteralConstant;
+import org.hypertrace.core.query.service.api.Value;
+import org.hypertrace.core.query.service.api.ValueType;
+
+/**
+ * Utility methods to easily create {@link org.hypertrace.core.query.service.api.QueryRequest} its
+ * selections and filters.
+ */
+public class QueryRequestUtil {
+
+  public static Expression createColumnExpression(String columnName) {
```

Comment:
Not too worried about test coverage on these guys - by their nature, to verify them, we'd be effectively duplicating the code in the test anyway.